features/upgrade/machines/machines.feature

```diff
@@ -37,9 +37,31 @@ Feature: Machine-api components upgrade tests
     | cluster_operator           |
     | "machine-api"              | # @case_id OCP-22712
     | "cluster-autoscaler"       | # @case_id OCP-27664
-    | "cloud-controller-manager" | # @case_id OCP-43331
 
 
+  # @author huliu@redhat.com
+  @upgrade-check
+  @admin
+  @4.10 @4.9
+  Scenario Outline: Cluster operator should be available after upgrade
```

Comment:
Only one case, seems don't need `Scenario Outline`, but it’s ok for me, look at others opinions.

Reply:
It is ok , I think , it will be easier to add other operators in the future if needed